### PR TITLE
add task loyout_product-card

### PR DESCRIPTION
**Pull request by volodymyrsar**
- [DEMO LINK](https://volodymyrsar.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://volodymyrsar.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/button.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/styles/button.css:5]: Move to the * selector and remove it from all other places.

File: src/styles/card.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+.image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  display: flex;
+  font-family: "Roboto", sans-serif;
```
Comment (OleziO) [inline src/styles/card.css:21]: Move to html selector and remove from all other places

**Review by OleziO — CHANGES_REQUESTED** [review]
Fix all comments.

File: src/styles/card.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  display: flex;
+  font-style: normal;
```
Comment (sTorba24) [inline src/styles/card.css:21]: it's default value, can  be removed

File: src/styles/card.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px;
+  color: var(--color-word);
+}
+
+.card__code {
+  font-style: normal;
```
Comment (sTorba24) [inline src/styles/card.css:30]: remove

File: src/styles/price.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.price__word {
+  font-style: normal;
```
Comment (sTorba24) [inline src/styles/price.css:8]: default value

File: src/styles/price.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--color-word2);
+}
+
+.price__value {
+  font-style: normal;
```
Comment (sTorba24) [inline src/styles/price.css:17]: remove

**Review by sTorba24 — APPROVED** [review]
Well done!

**Pull request closed without merging**